Community/Service Requests/service_request_host/templates/service_request_host_page.html

```diff
@@ -0,0 +1,45 @@
+<!-- Copyright 2018 BlueCat Networks (USA) Inc. and its affiliates
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+By: BlueCat Networks
+Date: 04-05-18
+Gateway Version: 18.6.1
+Description: Example Gateway workflows -->
+
+{% extends "base.html" %}
+
+{% block css %}
+<link rel="stylesheet" href="css/service_request_host_page.css?version={{ g.version | urlencode }}">
+{% endblock %}
+
+{% block title %}
+{{ text['title'] }}
+{% endblock %}
+∏
+{% block heading %}
+{{ text['title'] }}
+{% endblock %}
+
+{% block custom %}
+    <div id="deploy_status"></div>
+    <p>{{ text['info'] }}</p>
+    {% from "form_helper.html" import render_form %}
+    {{ render_form(form, action=url_for('service_request_hostservice_request_host_page_form'), id="service_request_host_form") }}
+
+    <input type="hidden" id="status_token" value="{{ status_token }}">
```

Comment:
I don't see anything in the workflow that sets the status_token when rendering the html template.

Reply:
This one I'm not really sure about. I took a look at my local files and it's not there. So either I caught it and it never got updated, or something very strange happened. Either way, I got it taken care of!